pages/p/[id].tsx

```diff
@@ -4,18 +4,25 @@ import prisma from '../../lib/prisma';
 import Layout from "../../components/Layout"
 import { PostProps } from "../../components/Post"
 import Error from 'next/error'
+import { auth } from '../../auth'
+import { Session } from 'next-auth'
 
-interface prismaError {
+interface PropObj {
```

Comment:
created a type object that has postprops , session object from authjs, and error's key values